packages/unlock-js/src/PublicLock/v4/updateRefundPenalty.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Update the name of a lock
+ * @param {object} params
+ * - {PropTypes.address} lockAddress : address of the lock for which we update the name
+ * - {string} name: the new name of the lock
+ * @param {function} callback invoked with the transaction hash
+ */
+export default async function (
+  { lockAddress, freeTrialLength, refundPenaltyBasisPoints },
+  transactionOptions = {},
+  callback
+) {
+  const lockContract = await this.getLockContract(lockAddress, this.provider)
+  const transactionPromise = lockContract.updateRefundPenalty(
```

Comment:
Can you quickly verifies that it works like this for all versions?

Reply:
This have same firm in abis, also tested in Goerli (v10/v11), and in Mainnnet (v8, but readonly because not lock manager for the tested locks, value returned are correct)